proposals/NNNN-Adding-Audible-Status-Notification.md

```diff
@@ -0,0 +1,66 @@
+# Adding Audible Status Notification
+
+* Proposal: [SDL-NNNN](NNNN-Adding-Audible-Status-Notification.md)
+* Author: [Shohei Kawano](https://github.com/Shohei-Kawano)
+* Status: **Create**
+* Impacted Platforms: [Core / iOS / Java Suite / RPC /HMI]
+
+## Introduction
+
+Currenlty, there is no way to independently notify the Audible status to applications that output audio such as SDL Media.  
+This time, `OnAudibleStatus` is added as a new API so that the status of Audible can be notified.
+
+
+## Motivation
+
+As a means to notify the Audible status to applications that output audio.  
+In the case of Mobile from Core, Audible without changing the HMI Level by using the parameters `audioStreamingState` and hmiLevel in RPC `OnHMIStatus` Can be notified.  
+However, in the case of Core from HMI, there is no appropriate API to notify the Audible status by itself,  
```

Comment:
Same question as above - can you please provide clarification regarding "Core from HMI" here?

Reply:
It mean "Notification from HMI to Core in HMI_API"

Reply:
```suggestion
However, there is currently no notification from HMI to Core to set the Audible status,
```